### PR TITLE
Enlighten App Information Added

### PR DESCRIPTION
**Pull request by devoirtechsandip**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

I have submitted a Xamarin form app which is live on the play store.

https://play.google.com/store/apps/details?id=com.devoirtech.enlighten

### Possible platform improvements

Its great.. :)

### CODE REVIEW

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
   <ItemGroup>
     <Folder Include="Views\Snippets\" />
-    <Folder Include="Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj:17]: Please undo the changes in this file

File: src/Snppts/Snippets/Enlighten.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+namespace Snppts.Snippets
+{
+    
+    public class Enlighten : IAmASnippet
+    {
+        public string Slug => "Enlighten";
+        public string Title => "Enlighten";
+        public string GithubRepoName => "devoirtechsandip/SnpptsChallenge";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
```
Comment (jfversluis) [inline src/Snppts/Snippets/Enlighten.cs]: I think this is Android only, right?

**Review by jfversluis — CHANGES_REQUESTED** [review]
Couple of small things, thanks!

File: src/Snppts/Snppts.csproj (hunk 1 above)
Comment (devoirtechsandip) [reply src/Snppts/Snppts.csproj:17]: Changes done. :)

File: src/Snppts/Snippets/Enlighten.cs (hunk 2 above)
Comment (devoirtechsandip) [reply src/Snppts/Snippets/Enlighten.cs]: Yes, and I have changed the file accordingly

**Review by jfversluis — APPROVED** [review]
Great! Thanks for your contribution!

**Comment by sthewissen** [comment]
Thanks for contributing to Snppts! Please fill in the following survey to receive your swag :)

https://www.surveymonkey.com/r/88ZQ6G2
